.pre-commit-config.yaml

```diff
@@ -0,0 +1,31 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.0.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: check-json
+    -   id: check-added-large-files
+
+#-   repo: https://github.com/pre-commit/mirrors-pylint
+#    rev: v2.3.1
+#    hooks:
+#    -   id: pylint
+#        name: pylint36
+#        language_version: python3.6
+
+#-   repo: https://github.com/pre-commit/mirrors-pylint
+#    rev: v1.9.1
+#    hooks:
+#    -   id: pylint
+#        name: pylint27
+#        language_version: python2.7
```

Comment:
Please remove commented code.

Reply:
I want to also add pylint, but lint is a bit more work, we could do that later, I'll remove those